packages/lib/config/getProjectConfig.ts

```diff
@@ -14,3 +14,11 @@ const PROJECT_CONFIG = process.env.NEXT_PUBLIC_PROJECT_ID
 export function getProjectConfig() {
   return PROJECT_CONFIG
 }
+
+export function isBeetsProject() {
+  return getProjectConfig().projectId === 'beets'
+}
+
+export function isBalancerProject() {
+  return getProjectConfig().projectId === 'balancer'
+}
```

Comment:
Can these not just be consts? 
```suggestion
export const isBeetsProject = getProjectConfig().projectId === 'beets'
export const isBalancerProject = getProjectConfig().projectId === 'balancer'
```

Reply:
yeah when you also implement the comment below -> updated